explorer/client/src/components/transaction-card/TransactionCard.tsx

```diff
@@ -114,20 +60,100 @@ function formatTxResponse(data: any) {
         },
         {
             label: 'Validator Signatures',
-            value: data.signatures,
+            value: tx.auth_sign_info.signatures,
             list: true,
             sublist: true,
         },
     ];
 }
 
-function TransactionCard({ txdata }: any) {
```

Comment:
@Jibz1 , let's make sure we use the SDK type if possible, this makes it much easier to debug type consistency